wrap/Makefile.serial

```diff
@@ -0,0 +1,216 @@
+
```

Comment:
What is the plan with these. Maintenance of two build systems will be a nightmare, and the version is already bumped to a new major version